jpa/subsystem/src/main/java/org/jboss/as/jpa/messages/JpaLogger.java

```diff
@@ -699,7 +699,7 @@
      * @return an {@link IllegalStateException} for the error.
      */
     @Message(id = 64, value =
-            "JTA transaction already has a 'SynchronizationType.UNSYNCHRONIZED' persistence context (EntityManager) joined to it " +
+            "Jakarta Transactions already have a 'SynchronizationType.UNSYNCHRONIZED' persistence context (EntityManager) joined to it " +
```

Comment:
Grammar: `Jakarta Transactions already have` should be `Jakarta Transaction already has`. It especially needs to be singular because the sentence ends with `joined to it` which is referring to the singular.

Reply:
made it `Jakarta Transactions transaction already has a 'SynchronizationType.UNSYNCHRONIZED' persistence context (EntityManager) joined to it`.